src/core/core.datasetController.js

```diff
@@ -492,19 +492,24 @@ helpers.extend(DatasetController.prototype, {
 		const iScale = meta.iScale;
 		const vScale = meta.vScale;
 		const stacked = isStacked(iScale, meta) || isStacked(vScale, meta);
-		let i, ilen, parsed;
+		let offset = 0;
+		let i, parsed;
 
-		if (helpers.isArray(data[start])) {
+		if (me.getDataset().parsed) {
```

Comment:
It'd be nice to make this an option at the chart level so that the user doesn't need to specify it for each individual dataset

I'm also wondering about the name. The user hasn't necessarily parsed anything, so I wonder if it's a bit of a misnomer. Maybe `dataStandardized`, `dataNormalized`, `parsingEnabled`, etc. might be other options, but they're all longer. I can't think of anything I really like

Reply:
How about `parsing: false` at chart level. Same key could be used for parsing options if there will be any.

Just for example, if earlier decimation would be implemented:
```js
parsing: {
  decimate: true
}

Reply:
Ok. That sounds reasonable to me